projects/table-builder/src/lib/components/header-menu/header-menu.component.html

```diff
@@ -22,7 +22,7 @@
 
       <ng-template #defaultFilter>
         <ng-container *ngSwitchCase="metaData.fieldType === FieldType.String || metaData.fieldType === FieldType.Array || metaData.fieldType === FieldType.Unknown || metaData.fieldType === FieldType.PhoneNumber">
-          <mat-form-field (click)="stopClickPropagate($event)" class="example-full-width font" style="padding: 0 16px">

```

Comment:
the method also prevented default - the directive only stops propagation

Reply:
why is prevent default important?
would it make sense to make a prevent-default directive?

Reply:
If you are changing code, you should make sure it is not important before removing.

Reply:
as far as I can tell it's behaving as expected. If you don't otherwise I will mark as resolved.